src/reducers/activity.js

```diff
@@ -6,15 +6,15 @@ import {
   type Action,
   DONE_FETCH_MESSAGES,
   MESSAGE_ADDED,
-  START_FETCH_MESSAGES,
+  START_FETCH_MESSAGES
 } from '../actions/activity';
 import type { ActivityLogMessage } from '../types/ActivityLogMessage';
 import { ACTIVITY_MESSAGES_LIMIT } from '../global/Constants';
 
 type State = {
   +messages: Array<ActivityLogMessage>,
   +isFetching: boolean,
-}
+};
```

Comment:
Still has changes :)

Reply:
They are not my changes

Reply:
@albertoestarrona @SeniorMobileDeveloper Whoever did the changes, can you please revert this. It's unnecessary and mess the PR with unrelated changes. Also, it will cause merge conflicts in future, so please.